xpra/codecs/nvenc/encoder.pyx

```diff
@@ -13,6 +13,8 @@ import ctypes
 from ctypes import cdll, POINTER
 from threading import Lock
 from pycuda import driver
+from pycuda._driver import LogicError
```

Comment:
Should be `from pycuda.driver import LogicError` or just refer to it as `driver.LogicError`.